bin/symfony_requirements

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env php
 <?php
 
-require_once dirname(__FILE__).'/../var/SymfonyRequirements.php';
```

Comment:
@lchrusciel what's going on here? this path is now wrong.

Reply:
https://github.com/sensiolabs/SensioDistributionBundle/issues/324

Reply:
My bad, on it